controllers/oauthAuthentication.js

```diff
@@ -30,7 +30,15 @@ export const oauthLoginAuthorize = (req, res) => {
 export const oauthLoginCallback = async (req, res, next) => {
   // check if the mandatory Authorization code is there
   if (!req.query.code) {
-    return res.sendStatus(400);
+    /**
+     * @throws the request is not authorized
+     * @see https://www.rfc-editor.org/rfc/rfc6749.html#section-4.1.2.1
+     */
+    const params = {
+      error: 'access_denied',
+      error_description: "vous avez invalidé la tentative d'identification, vous êtes maintenant déconnecté de ce service",
```

Comment:
can you explain to me the link between not providing query param `code` and this error description ?

Reply:
To keep it simple, OpenIDConnect cinematic. we check the way to do it and follow the principe of OpenID with Stephane. 
The code prove that you were Authenticated but in our case here, we were not. So there is no code. 
The case that you arrive here (login-callback) without code means that you reject the identification process.

Reply:
Mire needs to send error not the FS. but FS handle error page